core/db/migrate/20220113052823_create_payment_sources.rb

```diff
@@ -0,0 +1,19 @@
+class CreatePaymentSources < ActiveRecord::Migration[5.2]
+  def change
+    create_table :spree_payment_sources do |t|
+      t.string :gateway_source_id
```

Comment:
in spree_credit_cards we save that value in `gateway_payment_profile_id` . Should I use the same name here?

Reply:
@GrzegorzStochel let's use the same, it's going to be easier to remember this